quickstarts/microsoft.network/azurefirewall-create-with-ipgroups-and-linux-jumpbox/main.bicep

```diff
@@ -0,0 +1,430 @@
+@description('virtual network name')
+param virtualNetworkName string = 'vnet${uniqueString(resourceGroup().id)}'
+param ipgroups_name1 string = 'ipgroup1${uniqueString(resourceGroup().id)}'
+param ipgroups_name2 string = 'ipgroup2${uniqueString(resourceGroup().id)}'
+
+@description('Username for the Virtual Machine.')
+param adminUsername string
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Zone numbers e.g. 1,2,3.')
+param vmSize string = 'Standard_D2s_v3'
+
+@description('Number of public IP addresses for the Azure Firewall')
+@minValue(1)
+@maxValue(100)
+param numberOfFirewallPublicIPAddresses int = 1
+
+@description('Type of authentication to use on the Virtual Machine. SSH key is recommended.')
+@allowed([
+  'sshPublicKey'
+  'password'
+])
+param authenticationType string = 'sshPublicKey'
+
+@description('SSH Key or password for the Virtual Machine. SSH key is recommended.')
+@secure()
+param adminPasswordOrKey string
+
+var vnetAddressPrefix = '10.0.0.0/16'
+var serversSubnetPrefix = '10.0.2.0/24'
+var azureFirewallSubnetPrefix = '10.0.1.0/24'
+var jumpboxSubnetPrefix = '10.0.0.0/24'
+var nextHopIP = '10.0.1.4'
+var azureFirewallSubnetName = 'AzureFirewallSubnet'
+var jumpBoxSubnetName = 'JumpboxSubnet'
+var serversSubnetName = 'ServersSubnet'
+var jumpBoxPublicIPAddressName = 'JumpHostPublicIP'
+var jumpBoxNsgName = 'JumpHostNSG'
+var jumpBoxNicName = 'JumpHostNic'
+var jumpBoxSubnetId = resourceId('Microsoft.Network/virtualNetworks/subnets', virtualNetworkName, jumpBoxSubnetName)
+var serverNicName = 'ServerNic'
+var serverSubnetId = resourceId('Microsoft.Network/virtualNetworks/subnets', virtualNetworkName, serversSubnetName)
+var storageAccountName = '${uniqueString(resourceGroup().id)}sajumpbox'
+var azfwRouteTableName = 'AzfwRouteTable'
+var firewallName = 'firewall1'
+var publicIPNamePrefix = 'publicIP'
+var azureFirewallSubnetId = resourceId('Microsoft.Network/virtualNetworks/subnets', virtualNetworkName, azureFirewallSubnetName)
+var azureFirewallSubnetJSON = json('{"id": "${azureFirewallSubnetId}"}')
+var linuxConfiguration = {
+  disablePasswordAuthentication: true
+  ssh: {
+    publicKeys: [
+      {
+        path: '/home/${adminUsername}/.ssh/authorized_keys'
+        keyData: adminPasswordOrKey
+      }
+    ]
+  }
+}
+var networkSecurityGroupName = '${serversSubnetName}-nsg'
+var azureFirewallIpConfigurations = [for i in range(0, numberOfFirewallPublicIPAddresses): {
+  name: 'IpConf${i}'
+  properties: {
+    subnet: ((i == 0) ? azureFirewallSubnetJSON : json('null'))
+    publicIPAddress: {
+      id: resourceId('Microsoft.Network/publicIPAddresses', '${publicIPNamePrefix}${i + 1}')
+    }
+  }
+}]
+
+resource ipgroup1 'Microsoft.Network/ipGroups@2021-08-01' = {
+  name: ipgroups_name1
+  location: location
+  properties: {
+    ipAddresses: [
+      '13.73.64.64/26'
```

Comment:
what ip addresses are these?

Reply:
@vhorne - can you answer Brian's question?